com.unity.render-pipelines.universal/Runtime/2D/PixelPerfectCamera.cs

```diff
@@ -9,7 +9,9 @@ namespace UnityEngine.Experimental.Rendering.Universal
     [DisallowMultipleComponent]
     [AddComponentMenu("Rendering/2D/Pixel Perfect Camera (Experimental)")]
     [RequireComponent(typeof(Camera))]
-    [MovedFrom("UnityEngine.Experimental.Rendering.LWRP")] public class PixelPerfectCamera : MonoBehaviour, IPixelPerfectCamera
+    [MovedFrom("UnityEngine.Experimental.Rendering.LWRP")]
+    [HelpURL("https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest/index.html?subfolder=/manual/2d-pixelperfect.html%23properties")]
+    public class PixelPerfectCamera : MonoBehaviour, IPixelPerfectCamera
```

Comment:
I've just built the docs from this branch, and I can't figure out where is this link supposed to be rendered. I just don't see it. What is the expected output on the website? 
![image](https://user-images.githubusercontent.com/60658376/84652975-cfdc2180-af0c-11ea-9723-fae2769b6e59.png)

Reply:
This is for the help button on the header of the component:
![image](https://user-images.githubusercontent.com/2289361/84728199-49364b80-afc3-11ea-86cf-8562490a608b.png)